### PR TITLE
Conflicts resolved

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17157

## Description
Resolved conflicts. New branch due to inconsistency.

## Related PRs
https://github.com/demisto/content/pull/3918

### CODE REVIEW

File: Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 1]
```diff
@@ -3307,10 +3320,64 @@ def handle_msg(file_path, file_name, parse_only_headers=False, max_depth=3):
...
+                        if not isinstance(headers_map[header_key], list):
+                            # convert the existing value to array
+                            headers_map[header_key] = [headers_map[header_key]]
+
+                        # add the new value to the value array
+                        headers_map[header_key].append(header_value)
+                    else:
+                        headers_map[header_key] = header_value
+
+                header_words = header.split(' ')
```
Comment (ronykoz) [inline Scripts/ParseEmailFiles/ParseEmailFiles.py]:
```suggestion
                header_words = header.split(' ', 1)
```
Would make the following line redundant

File: Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 2]
```diff
@@ -3307,10 +3320,64 @@ def handle_msg(file_path, file_name, parse_only_headers=False, max_depth=3):
...
+        'CC': msg_dict['CC'],
+        'From': msg_dict['From'],
+        'Subject': headers_map['Subject'],
+        'HTML': msg_dict['HTML'],
+        'Text': msg_dict['Text'],
+        'Headers': headers,
+        'HeadersMap': headers_map,
+        'Attachments': '',
+        'Format': format_string,
+        'Depth': MAX_DEPTH_CONST - max_depth
```
Comment (ronykoz) [inline Scripts/ParseEmailFiles/ParseEmailFiles.py]: are you sure this line is correct?

File: Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 3]
```diff
@@ -3307,10 +3320,64 @@ def handle_msg(file_path, file_name, parse_only_headers=False, max_depth=3):
...
+
+                header_key = header_words[0][:-1]
+                header_value = ' '.join(header_words[1:])
+                header_value = header_value[:-1] if header_value[-1:] == ' ' \
+                    else header_value
+
+            else:
+                header_value += header[:-1] if header[-1:] == ' ' else header
+
+    email_data = {
```
Comment (ronykoz) [inline Scripts/ParseEmailFiles/ParseEmailFiles.py]: Did you check that all the needed data is here for the following functions?

Comment (teizenman) [reply Scripts/ParseEmailFiles/ParseEmailFiles.py]: These are objects that needs to be in every .MSG email

File: Scripts/ParseEmailFiles/ParseEmailFiles.py (hunk 2 above)
Comment (teizenman) [reply Scripts/ParseEmailFiles/ParseEmailFiles.py]: Its the same as in line 3583, where the original version of this code was written

**Pull request merged**